lightning/src/util/config.rs

```diff
@@ -271,6 +276,7 @@ impl_writeable_tlv_based!(ChannelConfig, {
 	(4, announced_channel, required),
 	(6, commit_upfront_shutdown_pubkey, required),
 	(8, forwarding_fee_base_msat, required),
+	(10, should_autoclose, required),
```

Comment:
This should be odd and have a `(default_value, X)` - no need to make old clients refuse to read new configs.